README.md

```
@@ -222,7 +224,28 @@ pnpm db:push       # Push schema directly (development only)
+
+1. **Create a Resend Account**:
+   - Sign up at [resend.com](https://resend.com)
+   - Create an API key in the dashboard
+
+2. **Verify Your Domain**:
+   - Add your sending domain in Resend dashboard
+   - Configure DNS records (SPF, DKIM) as instructed
+   - Use a verified domain email as `RESEND_EMAIL_FROM`
+
+3. **Set Environment Variables**:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)